app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -15,6 +15,7 @@ def create
       StatsD.increment("#{stats_key}.success")
       Rails.logger.info "ClaimID=#{claim.id} RPO=#{claim.education_benefits_claim.region} Form=#{form_type}"
       load_user
+      claim.after_submit(@current_user)
```

Comment:
This seems like a perfect use-case for an asynchronous sidekiq job.

Reply:
So the wording of the story gave me the indication that email should be sent immediately after form submission.  

Also the process is modifying a form property for use within in the spool files that are generated on a daily basis so this job would have always fire and finish before the `CreateDailySpoolFiles` sidekiq job